packages/javascript/bh-shared-ui/package.json

```diff
@@ -28,14 +28,16 @@
         "dompurify": "^2.3.0",
         "downshift": "^6.1.7",
         "js-client-library": "workspace:*",
+        "lodash": "^4.17.21",
         "luxon": "^3.3.0",
         "memoize-one": "^6.0.0",
         "notistack": "^2.0.5",
         "react-hook-form": "^7.45.1",
         "react-markdown": "^6.0.3",
         "react-query": "^3.39.3",
         "react-window": "^1.8.9",
-        "react-window-infinite-loader": "^1.0.9"
+        "react-window-infinite-loader": "^1.0.9",
+        "swagger-ui-react": "^4.15.2"
```

Comment:
Im getting unresolved dependency warnings while building `bh-shared-ui` from some of the packages imported into `OperationsEditionPlugin`. It looks like those modules are getting imported via sub-dependency's which is a little sketchy (this was happening before your changes btw, they just weren't caught via rollup). I think we aren't seeing errors because those dependencies exist via sub-dependencies right now so they resolve at run time!

I know Swaggers life in our app is limited, but It would be nice to not depend on those sub-dependencies existing for these docs to work in the meantime!

Can we add these to bh-shared-ui `package.json` dependencies:
- `immutable`, 
- `prop-types`, 
- and `react-immutable-proptypes`?

And in `rollup.config.js`, can we add to the external list: 
- `swagger-ui-react`, 
- `swagger-ui-react/swagger-ui.css`, 
- `prop-types`, 
- `immutable`, 
- `react-immutable-proptypes`, 
- and `lodash/toString`? 

That should resolve the dependency warning and also not rely on the subdependencys of other modules for `OperationsEditionPlugin`!